<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
  <channel>
    <title>OpenAI 开发者论坛 - 最新帖子</title>
    <link>https://community.openai.com</link>
    <description>最新帖子</description>
    <lastBuildDate>Sun, 31 Mar 2024 01:17:38 GMT</lastBuildDate>
    <item>
      <title>垃圾邮件新聊天以提高排名</title>
      <link>https://community.openai.com/t/spamming-new-chats-to-boost-ranking/689632#post_17</link>
      <description><![CDATA[他们是否向您提供了有关他们计划如何解决该问题的任何见解？鉴于您在前 12 名中拥有 3 个 GPT，我认为他们已经联系了您。不过，如果他们甚至不关心您，我也不会感到惊讶。]]></description>
      <guid>https://community.openai.com/t/spamming-new-chats-to-boost-ranking/689632#post_17</guid>
      <pubDate>Sun, 31 Mar 2024 01:13:11 GMT</pubDate>
    </item>
    <item>
      <title>使用API​​查询“gpt-4-0125-preview”</title>
      <link>https://community.openai.com/t/using-the-api-to-query-gpt-4-0125-preview/702634#post_7</link>
      <description><![CDATA[哦，更清楚一点：我的订阅仍然显示为“开启”，但他们似乎改变了计费方式......？]]></description>
      <guid>https://community.openai.com/t/using-the-api-to-query-gpt-4-0125-preview/702634#post_7</guid>
      <pubDate>Sun, 31 Mar 2024 01:07:00 GMT</pubDate>
    </item>
    <item>
      <title>使用API​​查询“gpt-4-0125-preview”</title>
      <link>https://community.openai.com/t/using-the-api-to-query-gpt-4-0125-preview/702634#post_6</link>
      <description><![CDATA[谢谢。实际上，只是问这个问题就让我产生了仔细检查这些日志并发现问题的想法。我的错。谢谢！]]></description>
      <guid>https://community.openai.com/t/using-the-api-to-query-gpt-4-0125-preview/702634#post_6</guid>
      <pubDate>Sun, 31 Mar 2024 01:06:09 GMT</pubDate>
    </item>
    <item>
      <title>在 Ubuntu 中导入 openai 时出错</title>
      <link>https://community.openai.com/t/error-while-importing-openai-in-ubuntu/702738#post_2</link>
      <description><![CDATA[您没有理由使用最先进的 Alpha 版本的 Python。
  &lt;标题类=“来源”&gt;
      
Python.org 上的讨论 – 23 日 10 月 30 日


  &lt;文章类=“onebox-body”&gt;
    
Python 3.13 alpha 1 包含重大更改，计划是什么？




核心开发






c-api





大家好，自从 CPython（遗憾地）离开 python-dev 以来，我并没有过多关注 CPython 的核心讨论，因此我遇到了针对 3.13a1 所做的 C-API 更改，几乎没有事先警告。至少变化的剪切量让我感到困惑......

阅读时间：7 分钟🕑
喜欢：131 ❤






尝试Python 3.10。]]></description>
      <guid>https://community.openai.com/t/error-while-importing-openai-in-ubuntu/702738#post_2</guid>
      <pubDate>Sun, 31 Mar 2024 01:03:29 GMT</pubDate>
    </item>
    <item>
      <title>我们如何保护自己免受骗子的侵害？</title>
      <link>https://community.openai.com/t/how-do-we-protect-ourselves-from-scam-artists/702741#post_2</link>
      <description><![CDATA[很明显，垃圾邮件发送者就是“钱包”网站。因此，有人永远不可信任，他是黄鼠狼，向 GPT 用户发送垃圾邮件反馈，值得被平台禁止。]]></description>
      <guid>https://community.openai.com/t/how-do-we-protect-ourselves-from-scam-artists/702741#post_2</guid>
      <pubDate>Sun, 31 Mar 2024 00:28:08 GMT</pubDate>
    </item>
    <item>
      <title>代理和心灵社会的概念</title>
      <link>https://community.openai.com/t/the-concept-of-agency-and-society-of-minds/702380#post_4</link>
      <description><![CDATA[代理人工智能现在是一个非常热门的话题，很多研究人员都在研究它。我认为 Andrew Ng 最近声称/发现多个 ChatGPT 3.5 代理协作分配任务、检查彼此的工作并迭代他们的创作，能够创建与 ChatGPT 4 相同级别的“输出”。 
有点像两个大脑比每个大脑的综合能力更聪明。
Andrew Ng 公布令人惊叹的人工智能架构 | “人工智能代理工作流程将推动人工智能的巨大进步”
人工智能先锋展示人工智能代理的力量 - “未来就是代理”]]></description>
      <guid>https://community.openai.com/t/the-concept-of-agency-and-society-of-minds/702380#post_4</guid>
      <pubDate>Sat, 30 Mar 2024 23:53:24 GMT</pubDate>
    </item>
    <item>
      <title>RateLimitError 仍然发生</title>
      <link>https://community.openai.com/t/ratelimiterror-still-occuring/702745#post_1</link>
      <description><![CDATA[嗨，
我几天前创建了我的 API 密钥。
我正在研究本教程。
我的所有使用次数都是 0，但我仍然收到此错误：
RateLimitError：错误代码：429 - {‘error’：{‘message’：‘您超出了当前配额，请检查您的计划和账单详细信息。有关此错误的更多信息，请阅读文档：link_here.&#39;, &#39;type&#39;: &#39;insufficient_quota&#39;, &#39;param&#39;: None, &#39;code&#39;: &#39;insufficient_quota&#39;}}
我看到很多人都受到这个问题的困扰，您可以在此处查看。
openAI的大佬能给个解释和解决方案吗？
（网站不允许我发布链接，所以我不得不删除它们）]]></description>
      <guid>https://community.openai.com/t/ratelimiterror-still-occuring/702745#post_1</guid>
      <pubDate>Sat, 30 Mar 2024 23:46:51 GMT</pubDate>
    </item>
    <item>
      <title>我们如何保护自己免受骗子的侵害？</title>
      <link>https://community.openai.com/t/how-do-we-protect-ourselves-from-scam-artists/702741#post_1</link>
      <description><![CDATA[所以你通过 OpenAI 收到像这样的电子邮件，你可以信任谁？：
你好，
您已收到有关 JukeBox 的反馈！ [[我的电子邮件]]写道：
您好，很乐意支持这项服务。您有 patreon、[gptwallets.co] 或其他东西吗？
XXX
如果您不想再收到这些电子邮件，可以在构建者个人资料中取消选中“接收反馈电子邮件”。
最好，
ChatGPT 团队]]></description>
      <guid>https://community.openai.com/t/how-do-we-protect-ourselves-from-scam-artists/702741#post_1</guid>
      <pubDate>Sat, 30 Mar 2024 23:29:29 GMT</pubDate>
    </item>
    <item>
      <title>助理线程可以存储在我们的服务器上而不是 OpenAI 上吗</title>
      <link>https://community.openai.com/t/can-assistant-threads-be-stored-on-our-server-instead-of-at-openai/697528#post_14</link>
      <description><![CDATA[在我看来，开发自己的 RAG + 聊天工作流程具有不可估量的优势：我不依赖提供商进行嵌入、文本完成等。我今天和明天都可以使用 OpenAI Claude，或者任何本地 AI解决方案。
我认为，如今人工智能领域/市场发生了如此频繁的变化……与单一提供商联姻并不是一个好的解决方案。
我想，当 Sam Altman 离开公司“几个小时”时，你们中的大多数人都在担心当前已经部署的人工智能技术堆栈的未来。小心这一点！]]></description>
      <guid>https://community.openai.com/t/can-assistant-threads-be-stored-on-our-server-instead-of-at-openai/697528#post_14</guid>
      <pubDate>Sat, 30 Mar 2024 23:28:40 GMT</pubDate>
    </item>
    <item>
      <title>在 Ubuntu 中导入 openai 时出错</title>
      <link>https://community.openai.com/t/error-while-importing-openai-in-ubuntu/702738#post_1</link>
      <description><![CDATA[嗨，
使用的版本：
Ubuntu (Jammy 22.04.3 LTS)
Python 3.13.0a5+
IDE Visual Studio Code
OpenAI：版本：1.14.3
已满足要求： ./mynewenv/lib/python3.13/site-packages (2.6.4) 中的 pydantic
已满足要求：.​​/mynewenv/lib/python3.13/site-packages 中的 annotated-types&gt;=0.4.0（来自 pydantic）（0.6.0）
已满足要求：.​​/mynewenv/lib/python3.13/site-packages 中的 pydantic-core==2.16.3（来自 pydantic）(2.16.3)
已满足要求：typing-extensions&gt;=4.6.1 in ./mynewenv/lib/python3.13/site-packages (来自 pydantic) (4.10.0)
运行main.py时出错
以下是编译main.py时的回溯
文件“/home/rg/MyStuff/laws_of_spirituality/main.py”，第 3 行，
导入openai
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/openai/init.py”，第 8 行，位于 
从 。导入类型
文件“/home/rg/MyStuff/laws_oFf_spirituality/mynewenv/lib/python3.13/site-packages/openai/types/init.py”，第 5 行，位于 
从.image导入图像作为图像
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/openai/types/image.py”，第 5 行，位于 
从…_models导入BaseModel
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/openai/_models.py”，第 590 行，位于 
类 FinalRequestOptions(pydantic.BaseModel):
...&lt;59 行&gt;...
model_construct = 构造
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/pydantic/_internal/_model_construction.py”，第 178 行，new
set_model_fields(cls, 基础, config_wrapper, types_namespace)
~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/pydantic/_internal/_model_construction.py”，第 452 行，在 set_model_fields
字段，class_vars =collect_model_fields（cls，基础，config_wrapper，types_namespace，typevars_map = typevars_map）
~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/pydantic/_internal/_fields.py”，第214行，在collect_model_fields
field_info = FieldInfo.from_annotated_attribute(ann_type, 默认)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^ ^^^
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/pydantic/fields.py”，第 372 行，from_annotated_attribute
if _typing_extra.is_annotated(注释):
~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/pydantic/_internal/_typing_extra.py”，第 103 行，is_annotated
返回原点不是 None 且 lenient_issubclass(origin, Annotated)
~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
文件“/home/rg/MyStuff/laws_of_spirituality/mynewenv/lib/python3.13/site-packages/pydantic/_internal/_utils.py”，第 76 行，位于 lenient_issubclass
返回 isinstance(cls, type) 和 issubclass(cls, class_or_tuple)
~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^
文件“/usr/local/lib/python3.13/typing.py”，第 501 行，subclasscheck
raise TypeError(f&quot;{self} 不能与 issubclass() 一起使用&quot;)
类型错误：typing.Annotated 不能与 issubclass() 一起使用]]></description>
      <guid>https://community.openai.com/t/error-while-importing-openai-in-ubuntu/702738#post_1</guid>
      <pubDate>Sat, 30 Mar 2024 23:18:34 GMT</pubDate>
    </item>
    <item>
      <title>您好，我想在 ChatCompletion 创建 API 中传递多个提示</title>
      <link>https://community.openai.com/t/hi-i-want-to-pass-multiple-prompts-in-chatcompletion-create-api/698428#post_16</link>
      <description><![CDATA[同时发出大量请求可能会提前达到速率限制。每分钟请求数也可以视为类似于每秒请求数 * 60。
这提供了一个框架，用于向 API 发出速率受限的并行请求，手动指定端点而不是使用 openai 库。

  &lt;标题类=“来源”&gt;

      github.com


  &lt;文章类=“onebox-body”&gt;
    openai/openai-cookbook/blob /main/examples/api_request_parallel_processor.py
&quot;&quot;&quot;
API 请求并行处理器

使用 OpenAI API 快速处理大量文本需要小心。
如果您一一滴入一百万个 API 请求，它们将需要几天的时间才能完成。
如果您并行淹没一百万个 API 请求，它们将超出速率限制并因错误而失败。
为了最大化吞吐量，需要限制并行请求以保持在速率限制之下。

该脚本并行化对 OpenAI API 的请求，同时进行限制以保持在速率限制之下。

特征：
- 流式传输文件请求，以避免大型作业内存不足
- 同时发出请求，以最大化吞吐量
- 限制请求和令牌使用，以保持在速率限制之下
- 重试失败的请求最多 {max_attempts} 次，以避免丢失数据
- 记录错误，以诊断请求问题

调用脚本的示例命令：
````
python 示例/api_request_parallel_processor.py \




  该文件已被截断。 显示原始内容





它可以变得更加简单，并围绕 openai 库编写，专注于使用系统消息完成聊天，该系统消息对多个输入执行相同范围的操作，并使用简单的输入文件和可恢复的响应磁盘数据库。还可以改进以获取 x-ratelimit 标头并在获得 API 拒绝之前调整限制。]]></description>
      <guid>https://community.openai.com/t/hi-i-want-to-pass-multiple-prompts-in-chatcompletion-create-api/698428#post_16</guid>
      <pubDate>Sat, 30 Mar 2024 23:13:35 GMT</pubDate>
    </item>
    </channel>
</rss>